ros2run/ros2run/command/run.py

```diff
@@ -22,7 +22,7 @@
 
 
 class RunCommand(CommandExtension):
-    """Various run related sub-commands."""
+    """Run a package specific executable."""
```

Comment:
A totally unambiguous way to say it is `Run an executable of a specific package.`